docs/hooks_and_conditions.md

```diff
@@ -7,19 +7,22 @@ set up conditions for when the method should fire.
 ## Decorator Signature
 
 ```python
-    @hook(
-        moment: str,
-        when: str = None,
-        when_any: List[str] = None,
-        has_changed: bool = None,
-        is_now: Any = '*',
-        is_not: Any = None,
-        was: Any = '*',
-        was_not: Any = None,
-        changes_to: Any = None,
-        priority: int = DEFAULT_PRIORITY,
-        on_commit: Optional[bool] = None
-    ):
+@hook(
+    moment: str,
+    condition: Optional[types.Condition] = None,
+    priority: int = DEFAULT_PRIORITY,
+    on_commit: Optional[bool] = None,
+    
+    # Legacy parameters
+    when: str = None,
+    when_any: List[str] = None,
+    has_changed: bool = None,
+    is_now: Any = '*',
```

Comment:
If you are using `"` shouldn't you use it everywhere?

Reply:
because ruff doesn't format .mds :facepalm: I will leave as it is until I can format it automatically